features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2818,6 +2818,8 @@
                     <Path>/api/identity/auth/v1.1</Path>
                     <Path>/api/server/v1/authenticators</Path>
                     <Path>/api/server/v1/branding-preference</Path>
+                    <Path>/api/server/v1/validation-rules</Path>
+                    <Path>/sessions</Path>
```

Comment:
This path is set in the form of `/api/users/v1/5b4620d2-ea08-464f-9d31-05b2abe2b41c/sessions`.  
But the  SubOrganizationContextRewriteValve check the sub paths using `contains` check as shown here https://github.com/wso2-extensions/identity-carbon-auth-rest/blob/master/components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/valve/OrganizationContextRewriteValve.java#L91.
Due to that, the sub path is configured in this format. `<Path>/sessions</Path>`